tests/run-tests-on-external-cluster.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+: {$KUBEVIRTNAMESPACE:="openshift-cnv"}
+_kubectl=$(which kubectl)
+_oc=$(which oc)
+
+[[ -z ${_kubectl} && -z ${_oc} ]] && echo "OC or KUBECTL is not installed" && exit 1
+
+[ -z "${KUBECONFIG}" ] && echo "KUBECONFIG is not set" && exit 1
+
+oc create namespace kubevirt
+rm -rf _out
+hack/dockerized hack/build-func-tests.sh
+make manifests
+rm -f _out/manifests/testing/cdi-*
+rm -f _out/manifests/testing/kubevirt-config.yaml
+sed -i "s/namespace: kubevirt/namespace: ${KUBEVIRTNAMESPACE}/g" _out/manifests/testing/*.yaml
```

Comment:
KUBEVIRTNAMESPACE here , but hardcoded on line 11, is it fine ?

Reply:
These do not seem to be the same. Lets continue the discussion on line 11 comments.